bin/iuf-skopeo-sync

```diff
@@ -50,13 +59,44 @@ process(){
     # Remove the leading path element. 
     ns_path=$(echo $docker_dir|cut -d'/' -f2-)
 
-    skopeo sync --scoped \
+    if skopeo sync --scoped \
         --src dir --dest docker \
         --dest-creds "${NEXUS_USERNAME:-admin}:${NEXUS_PASSWORD}" \
         --dest-tls-verify=false \
-        /images/"$docker_dir" ${NEXUS_REGISTRY:-registry.local}/$ns_path
+        /images/"$docker_dir" ${NEXUS_REGISTRY:-registry.local}/$ns_path; then
+
+        # Record results for each image. Note that we have to search the directory for
+        # things that appear to be docker images here, which unfortunately is not directly
+        # looking at the result of the `skopeo sync` command.
+        find "/images/$docker_dir" -name "manifest.json" | while read -r manifest_file; do
+            image_name_and_version=$(basename "$(dirname "$manifest_file")")
+            image_name="$(cut -d":" -f 1 <<< "$image_name_and_version")"
+            image_version="$(cut -d":" -f 2 <<< "$image_name_and_version")"
+            echo "INFO recording results for ${image_name}:${image_version}"
+            record_result "$image_name" "$image_version"
+        done
+
+    fi
+
 }
 
+# usage: record_result IMAGE_NAME IMAGE_VERSION
+#
+# Writes part of the results to the $RESULTS_FILE.
+#
+function record_result(){
+    # append records for a single repository to the results file.
+    image_name="$1"
+    image_version="$2"
```

Comment:
Just a note that this could potentially silently create a corrupt results file if either argument is missing. Not sure if it's a big deal here, but something to be aware of.

Reply:
I could consider putting in default values, at least to avoid inserting incorrect data. Probably something to consider for the other script as well.

Reply:
Just to quickly follow up on this, as it currently exists what would end up happening is a failure to insert data due to incompatible data, as we validate this data against the schema. I think that's probably acceptable.